activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -29,6 +30,23 @@ def test_dump_schema_information_outputs_lexically_ordered_versions
     ActiveRecord::SchemaMigration.delete_all
   end
 
+  if current_adapter?(:SQLite3Adapter)
+    %w{3.7.8 3.7.11 3.7.12}.each do |version_string|
+      test "dumps schema version for sqlite version #{version_string}" do
+        stub_version(version_string) do
```

Comment:
I had tried this, but `SQLite3Adapter::Version` isn't available here, so fell back to how we use it in other tests. 
I will revise this.